doc/redirects.json

```diff
@@ -1950,21 +1983,6 @@
   "ex-dockerTools-streamNixShellImage-addingShellHook": [
     "index.html#ex-dockerTools-streamNixShellImage-addingShellHook"
   ],
-  "sec-pkgs-ociTools": [
-    "index.html#sec-pkgs-ociTools"
-  ],
-  "ssec-pkgs-ociTools-buildContainer": [
-    "index.html#ssec-pkgs-ociTools-buildContainer"
-  ],
-  "ssec-pkgs-ociTools-buildContainer-inputs": [
-    "index.html#ssec-pkgs-ociTools-buildContainer-inputs"
-  ],
-  "ssec-pkgs-ociTools-buildContainer-examples": [
-    "index.html#ssec-pkgs-ociTools-buildContainer-examples"
-  ],
-  "ex-ociTools-buildContainer-bash": [
```

Comment:
Please don't delete anchors but create redirects by moving those old locations into the lists of the new identifiers. That way users who have bookmarked these will automatically be pointed to the new thing instead of to nowhere.


Also for such a substantial change, please add a release note. I'd even consider leaving the old interface in place with a deprecation warning, because simply removing it will break setups that rely on it.

Reply:
I added a release note and a warning. Or do you mean an explicit note regarding buildContainer?

Reply:
You've added an error not a warning, which forces users to migrate on update rather than giving a release cycle worth of advance time, which we typically do with changes of such scope.

For the documentation, this is the same. One thing you could do is take all the removed locations (the items within the lists that have been deleted) and gather them under a new identifier (the key in this JSON) that is referred to from the release note (see the other comment).

Reply:
> You've added an error not a warning, which forces users to migrate on update rather than giving a release cycle worth of advance time, which we typically do with changes of such scope.

Fair. I'll re-introduce the `buildContainer` interface with a warning and reinstantiate the relevant old doc sections, also with a warning. In that case, I'd not even add a relocation for these yet. Are you fine with this? Or would you not keep the old documentation at all and instead have relocations?

Reply:
Yes, leaving the old docs and leaving a note that this interface is deprecated is fine as well!